contracts/LiquidDemocracy.sol

```diff
@@ -56,6 +57,59 @@ contract LiquidDemocracy {
 
         votersData[voterAddress].weight = 1;
         votersData[voterAddress].registered = true;
+        delegations[voterAddress] = voterAddress;
+    }
+
+    // Delegations
+
+    /**
+     * Delegate the voting power of a voter to another voter
```

Comment:
Let's add the `@notice` keyword to comply with [Natspec](https://github.com/ethereum/wiki/wiki/Ethereum-Natural-Specification-Format), also true for `getMyRepresentative()` and `getMyWeight()`

Reply:
I'm not sure who came up with the brilliant idea of having to write a redundant keyword for every  description of a function :P . But I agree, let's just stick to a standard.

Reply:
lol